support/ext/apcu

```diff
@@ -4,8 +4,8 @@ set -e
 
 php_version=${1:=7.4}
 
-if echo $php_version | grep -q -E '^7.' ; then
-  apcu_version=5.1.18
+if echo $php_version | grep -q -E '^[78].' ; then
```

Comment:
I think our buildpack does not support PHP < 7. Hence I would remove completely the condition.

Reply:
Actually, our buildpack still support PHP 5.x on old stack, it's not available for our users anymore, but I would keep it in the buildpack since it works and it's an open-source project people may use (sometimes on legacy stacks)